### ISSUE TITLE
A few questions

### ISSUE THREAD

**Issue by aravind598** [bes-dev/MobileStyleGAN.pytorch#42]
Hi there, much thanks for your work on MobileStyleGAN I have a few questions to ask. 

1. How long should the training last?
I have been training on my custom StyleGAN2 Model for 70+ epochs and the original loss seems to be maintaining at around 7. How many epoch was the original MobileNet FFHQ student network trained for?

2. What are the losses to look out for?
```python
Epoch 2:  18%|▉    | 2546/14000 [59:37<4:28:15,  1.41s/it, loss=10.041, v_num=1234, l1=5.73, l2=4.59, loss_p=2.04, loss_g=7.92, d_reg=0.0189, loss_d=0.00057, kid_val=0.375, loss_val=18.9]
```
For the losses above what would be their target value roughly speaking and which of the losses should I take note of or is the most important in this case? Also what are the kid_val target for the FFHQ model in the paper? 

3.  Does the ```kid_val was not in the top True``` result affect training in any way?

I would much appreciate your reply. Thanks

Edit: Apologies for tagging @bes-dev

**Comment by RahulBhalley** [comment]
@aravind598 could you please share the `pip freeze` and python version you're using?
